test/_is_undici_incompat.js

```diff
@@ -29,7 +29,13 @@ function isUndiciIncompat() {
   const undiciVer = require('undici/package.json').version;
   const msg = `undici@${undiciVer} is incompatible with node@${nodeVer}`;
 
-  if (satisfies(undiciVer, '>=6.13.0') && satisfies(nodeVer, '<18.17.0')) {
+  if (satisfies(undiciVer, '>=7.0.0') && satisfies(nodeVer, '<20.18.1')) {
```

Comment:
Do you know the min-supported Node is 20.18.1 and not some other Node v20 version?
... ah, I see the update to their engines. I guess that is their prerogative, though I think that is pretty harsh for users.

Reply:
> ... though I think that is pretty harsh for users.

I agree. At 1st I thought there might be a plan to use a feature which requires that version but I haven't found any clue about it. so I guess basically encouraging users to update nodejs I guess